src/screens/Key/Create/CreateKeySuccessScreen/styles.js

```diff
@@ -0,0 +1,32 @@
+import { MediaQueryStyleSheet } from 'react-native-responsive';

```

Comment:
This file was removed from develop the same as the whole CreateKeySuccessScreen directory. It looks like merging error, please remove that directory.